tests/lifecycle/common_test.go

```diff
@@ -0,0 +1,30 @@
+// +build !unit
+
+package lifecycle
+
+import (
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/open-service-broker-azure/pkg/azure"
+)
+
+func getAzureConfigAndAuthorizer() (
```

Comment:
Merging `GetConfigFromEnvironment` and `GetBearerTokenAuthorizer` into one is a little strange, this function does too much thing.

Reply:
Got your point. Will separate them.